client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/test/dynamic-settings-form.js

```diff
@@ -1,121 +0,0 @@
-/** @format */
```

Comment:
Just renamed this to `client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/test/dynamic-settings-form.test.js` and converted a few tests to leverage Redux (since this is now a connected component)